app/views/users/show_follow.html.erb

```diff
@@ -0,0 +1,30 @@
+<% provide(:title, @title) %>
+<div class="row">
+  <aside class="col-md-4">
+    <section class="user_info">
+      <%= gravatar_for @user %>
+      <h1><%= @user.name %></h1>
+      <span><%= link_to "view my profile", @user %></span>
+      <span><b><%= t ".microposts" %></b> <%= @user.microposts.count %></span>
```

Comment:
nó không lấy các microposts từ CSDL.nó thực hiện tính toán trong CSDL.yêu cầu CSDL đếm các microposts với user_id
gọi chiều dài trên mảng bằng hàm count()
cái nay trở nên ko hiệu quả nếu microposts lớn